app/api/routes.py

```diff
@@ -17,25 +19,28 @@ def languages():
 
 
 def get_resources():
-    resources = {}
     try:
-        resources = Resource.query.all()
+        page = request.args.get('page', 1, type=int)
+        resource_paginator = Resource.query.paginate(page, Config.RESOURCES_PER_PAGE, False)
+        resource_list = [resource.serialize for resource in resource_paginator.items]
 
     except Exception as e:
         print_tb(e.__traceback__)
         print(e)
-
+        resource_list = []
     finally:
-        return jsonify([single_resource.serialize for single_resource in resources])
+        return jsonify(resource_list)
```

Comment:
We'll eventually want to start returning more than a json array. But this is good for now.

Reply:
Feel free to open an issue so that someone can jump on fixing that for all the routes. I'm not sure what else we ought to be returning. We're not rendering views from the API, are we?

Reply:
Things like status, or total number of resources, or number of total pages, or some type of cache reference. 

Like all these guys:
https://api.slack.com/methods/users.list

Reply:
Oh, right. Yeah, we'll probably want to add some of those.